README.md

```diff
@@ -971,10 +1251,25 @@ func (r *CairoRunner) RunUntilPC(end memory.Relocatable) error {
   }
  }
  return nil
-}
 ```
 
-Once we are done executing, we can relocate our memory & trace and output them into files
+*Step*
+
+```go
+    //TODO
+```
+
+*Decode instruction*
+
+```go
+    //TODO
+```
+
+*Run instruction*
+
+```go
+    //TODO
+```
```

Comment:
I think line 650 already has this. We can remove it.

Reply:
done